### ISSUE TITLE
SyntaxError: Unexpected token ...

### ISSUE THREAD

**Issue by lhtdesignde** [danethurber/webpack-manifest-plugin#193]
Hi, I just updated to version 2.1.0 and I'm getting this error when building:
`SyntaxError: Unexpected token ...`

My config:
```
        plugins: [
          new ManifestPlugin({
            fileName: '../../../.js-names.json',
            filter: (file) => !file.path.match(/\.map$|\d-chunk/)
          })
        ],
```

Guessing it has problems to handle es6 features. Any idea how I could  fix this?

**Comment by mastilver** [comment]
Hi @lhtdesignde 

Unfortunatly we dropped support for older node versions (see release note: https://github.com/danethurber/webpack-manifest-plugin/releases/tag/v2.0.0)

Please either pin your version of `webpack-manifest-plugin` or upgrade your node version

**Comment by samuelmeuli** [comment]
The release notes don't mention Node 6, which is specified in `package.json`: https://github.com/danethurber/webpack-manifest-plugin/blob/687e2608b6c48d9ce6f26ce03df02de8db7c5f9b/package.json#L13

**Comment by mastilver** [comment]
Yep, I just realised, I was looking array spread...

Indeed object spread is not available until 8.6

**Comment by lhtdesignde** [comment]
aah that explains a lot. Running node 6 here. Thanks!
